srtcore/group.cpp

```diff
@@ -3637,7 +3637,7 @@ void CUDTGroup::sendBackup_CheckParallelLinks(const vector<gli_t>& unstable,
             // Some sockets could have been closed in the meantime.
             if (m_SndEpolld->watch_empty())
             {
-                HLOGC(gslog.Debug, log << "grp/sendBackup: no more sendable sockets - group broken");
+                HLOGC(gslog.Debug, log << "grp/sendBackup: no more active sockets - group broken");
```

Comment:
Actually here you could leave the "sendable" name - as surprisingly here it means both active and idle links (likely a legacy from old copy from broadcast). It wouldn't make sense otherwise - you don't make the group broken only because of having no active links if you still have idle links you can activate.